packages/qwik-city/middleware/aws-lambda/index.ts

```diff
@@ -0,0 +1,69 @@
+import 'source-map-support/register';
+import serverless from 'serverless-http';
+import { createQwikCity as createQwikCityNode } from '@builder.io/qwik-city/middleware/node';
+import type { ServerRenderOptions } from '@builder.io/qwik-city/middleware/request-handler';
+
+/**
+ * @public
+ */
+export function createQwikCity(opts: any) {
+  try {
+    const { router, staticFile, notFound } = createQwikCityNode({
+      render: opts.render,
+      manifest: opts.manifest,
+      qwikCityPlan: opts.qwikCityPlan,
+      static: {
+        cacheControl: 'public, max-age=31557600',
+      },
+      getOrigin(req) {
+        if (process.env.IS_OFFLINE) {
+          return `http://${req.headers.host}`;
+        }
+        return null;
+      },
+    });
+
+    const qwikApp = serverless(
```

Comment:
Hi @leifermendez while testing out this change, i'm seeing an issue with intercepting query parameters on the main route '/', where I can't query is always null, but the pathname is interceptable, wondering if there's something else that's needed here to address this.